script/build

```diff
@@ -268,11 +268,24 @@ def main() -> None:
                 *cmake_args,
                 verbose=True,
             )
-        except subprocess.CalledProcessError as exc:
+        except subprocess.CalledProcessError:
             print_error("Build failed. If the output makes no sense, try these steps:")
             print_error("1. Delete '_skbuild' directory")
             print_error(f"2. Delete conan directory: {Path.home() / '.conan'}")
             sys.exit(1)
+        setup_scripts = ["src/clib/lib/job_queue/qstat_proxy.sh"]
+        try:
+            for script in setup_scripts:
+                script_base_name = os.path.basename(script)
+                run(
+                    "cp",
+                    script,
+                    os.path.join(os.path.dirname(sys.executable),
+                        script_base_name),
+                )
+        except subprocess.CalledProcessError:
+            print_error("Failed to copy script files to PATH")
+            sys.exit(1)
```

Comment:
Should replace with `shutil.copyfile`. Ie, don't need to run `/bin/cp` when we already have it builtin into python.

Reply:
good point, thanks

Reply:
no longer relevant, resolving this